benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/contention/benchmark/OpCounter.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.contention.benchmark;
+
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchOp;
+import org.apache.datasketches.kll.KllFloatsSketch;
```

Comment:
I continue my general comment:

2. As this is significant write up and code change, that can not be reviewed well enough to be convinced in its correctness, I suggest to make the following correctness checks:

- 2a. Run *all* benchmarks currently existing on master to see they do not fail or give some weird results after the change.

- 2b. For all benchmarks currently existing on master, run them in their previous version (master) and with this code and verify that the results are similar.